source/panel_predictor/__main__.py

```diff
@@ -43,7 +43,7 @@ def run_live(framework):
 
         if cv2.waitKey(1) & 0xFF == ord('q'):  # press q to quit
             break
-        ret, frame = cap.read()  # get next frame
+        frame = cap.read()  # get next frame
```

Comment:
Does this actually work? It's my understanding from [this documentation](https://docs.opencv.org/3.4/d8/dfe/classcv_1_1VideoCapture.html#a473055e77dd7faa4d26d686226b292c1) that cap.read() will return a tuple with two values

Reply:
cap.read() will return a tuple when used on a video stream. unfortunately, it only returns 1 from a live input. This was previously throwing an error

Reply:
We are also not using VideoCapture() in this function, we are using VideoStream(), which only has one return value https://github.com/jrosebr1/imutils/blob/master/imutils/video/videostream.py